cohortextractor/query_engines/mssql.py

```diff
@@ -62,7 +62,7 @@ class MSSQLDateTime(_MSSQLDateTimeBase, sqlalchemy.types.TypeDecorator):
 
 
 class MssqlQueryEngine(BaseSQLQueryEngine):
-    sqlalchemy_dialect = sqlalchemy.dialects.mssql
+    sqlalchemy_dialect = sqlalchemy.dialects.mssql.pymssql.dialect
```

Comment:
I think this means we can tighten BaseSQLQuery.sqlalchemy_dialect type to `type[Dialect]` now?